triciaswift/tastevine-server
# Added registration method

I added a register method to auth.py

Supported route:
`/register` for a POST request that returns a single object of the newly created token

***TO TEST***
1. In your terminal: `git fetch`
2. Checkout `ts-register` branch
3. Start debugger
4. Open Postman
5. Send a POST to `http://localhost:8000/register` with the following JSON body or something similar:
```
{
    "username": "megcharme",
    "email": "meg@ducharme.com",
    "password": "ducharme",
    "first_name": "Meg",
    "last_name": "Ducharme"
}
```
6. Should receive a 200 status and a similar looking response body:
```
{
    "token": "c8fb7595d4416fe2c7c62d8f550568153cdb798c"
}
```
